src/posit/connect/permissions.py

```diff
@@ -137,3 +148,86 @@ def get(self, uid: str) -> Permission:
         url = self.params.url + path
         response = self.params.session.get(url)
         return Permission(self.params, **response.json())
+
+    def destroy(self, *permissions: str | Group | User | Permission) -> list[Permission]:
+        """Destroy supplied content item permissions.
```

Comment:
```suggestion
        """Remove supplied content item permissions.
```

This might speak to our users more than `Destroy`?